### PR TITLE
Fix: extensions with multiple module-level eval expressions

### PR DESCRIPTION
**Pull request by jberdine**

### CODE REVIEW

File: test/passing/extensions.ml [hunk 1]
```diff
@@ -23,3 +23,10 @@ let invariant t =
        , xxxxxxxxxxxxxx
        , xxxxxxxxxxx
        , xxxxxxxxxxxxxxxxxxxx ) when a < b]
+
+[%%ext
+;; 11111111111111111111
+
+;; 22222222222222222222
+
+;; 33333333333333333333]
```
Comment (jberdine) [inline test/passing/extensions.ml]: @hhugo, does this fix the problematic case you ran into with #181 ?

Comment (hhugo) [reply test/passing/extensions.ml]: I believe so

**Comment by jberdine** [comment]
should fix #184

File: test/passing/extensions.ml (hunk 1 above)
Comment (hhugo) [reply test/passing/extensions.ml]: Could we special case when there is one item only?

Comment (jberdine) [reply test/passing/extensions.ml]: #195 ?

**Pull request merged**